frontend/src/hooks/authentication/useShowPassword.ts

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+export const useShowPassword = () => {
+  const [showPassword, setShowPassword] = useState(false);
+
+  const handlePasswordToggle = () => {
+    setShowPassword((prevShowPassword) => !prevShowPassword);
+  };
+
+  return {
+    showPassword,
+    handlePasswordToggle,
+  };
+};
```

Comment:
훅으로 분리해준거 너무 좋은 것 같아요! 👍 그런데 생각보다 범용적으로 사용할 수 있을 것 같아서 아래와 같이 조금 더 범용적인 이름으로 변경해봐도 좋을 것 같아요!! 

(저도 useDropdown 내부에서 이렇게 toggle 함수를 만들어서 사용하고 있는데, 이렇게 범용적인 훅으로 만들면 재사용할 수 있을 것 같네요!!)

```suggestion
export const useToggleOpen = (initState = false) => {
  const [isOpen, setIsOpen] = useState(initState);

  const toggleOpen = () => {
    setIsOpen((prev) => !prev);
  };

  return [isOpen, toggleOpen]
};
```

```
// 사용처
const [showPassword, handlePasswordToggle ] = useToggleOpen();
```